robottelo/constants/__init__.py

```diff
@@ -480,8 +490,8 @@
         'key': 'rhva6',
     },
     'rhva65': {
-        'name': ('Red Hat Enterprise Virtualization Agents for RHEL 6 Server RPMs x86_64 6.5'),
-        'version': '6.5',
+        'name': ('Red Hat Enterprise Virtualization Agents for RHEL 6 Server RPMs x86_64 6.8'),
+        'version': '6.8',
```

Comment:
@lhellebr is this right? I am guessing, `rhva65` is referring to rhel version here and not satellite tools. Need to check.

Reply:
I think Jameer is right, this refers to RHEL version, not Satellite. Newest RHEL 6 is 6.10.

Also, variable name `rhva65` should probably be changed as well (along with everything that might use this name...).

Reply:
Yes, this refers to RHEL version. Restored 6.5 and added entries for 6.10.